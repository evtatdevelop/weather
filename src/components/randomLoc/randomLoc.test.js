import React from 'react';
import RandomLoc from './randomLoc';
import renderer from 'react-test-renderer';
// import {shallow} from 'enzyme';

describe('Testing <RandomLok/>', () => {
  it('RandomLoc have rendered correctly', () => {
    const loc = renderer.create(<RandomLoc/>).toJSON();
    // const loc = shallow(<RandomLoc/>);
    expect(loc).toMatchSnapshot();
  })
});