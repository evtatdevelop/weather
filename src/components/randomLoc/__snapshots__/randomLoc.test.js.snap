// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <RandomLok/> RandomLoc have rendered correctly 1`] = `
<section
  className="module"
>
  <svg
    height="104px"
    preserveAspectRatio="xMidYMid"
    viewBox="0 0 100 100"
    width="104px"
    xmlns="http://www.w3.org/2000/svg"
    xmlnsXlink="http://www.w3.org/1999/xlink"
  >
    <g
      transform="rotate(0 50 50)"
    >
      <rect
        fill="#f5deb3"
        height="3"
        rx="1.5"
        ry="1.5"
        width="3"
        x="48.5"
        y="39.5"
      >
        <animate
          attributeName="opacity"
          begin="-2.1103896103896105s"
          dur="2.272727272727273s"
          keyTimes="0;1"
          repeatCount="indefinite"
          values="1;0"
        />
      </rect>
    </g>
    <g
      transform="rotate(25.714285714285715 50 50)"
    >
      <rect
        fill="#f5deb3"
        height="3"
        rx="1.5"
        ry="1.5"
        width="3"
        x="48.5"
        y="39.5"
      >
        <animate
          attributeName="opacity"
          begin="-1.948051948051948s"
          dur="2.272727272727273s"
          keyTimes="0;1"
          repeatCount="indefinite"
          values="1;0"
        />
      </rect>
    </g>
    <g
      transform="rotate(51.42857142857143 50 50)"
    >
      <rect
        fill="#f5deb3"
        height="3"
        rx="1.5"
        ry="1.5"
        width="3"
        x="48.5"
        y="39.5"
      >
        <animate
          attributeName="opacity"
          begin="-1.7857142857142856s"
          dur="2.272727272727273s"
          keyTimes="0;1"
          repeatCount="indefinite"
          values="1;0"
        />
      </rect>
    </g>
    <g
      transform="rotate(77.14285714285714 50 50)"
    >
      <rect
        fill="#f5deb3"
        height="3"
        rx="1.5"
        ry="1.5"
        width="3"
        x="48.5"
        y="39.5"
      >
        <animate
          attributeName="opacity"
          begin="-1.6233766233766234s"
          dur="2.272727272727273s"
          keyTimes="0;1"
          repeatCount="indefinite"
          values="1;0"
        />
      </rect>
    </g>
    <g
      transform="rotate(102.85714285714286 50 50)"
    >
      <rect
        fill="#f5deb3"
        height="3"
        rx="1.5"
        ry="1.5"
        width="3"
        x="48.5"
        y="39.5"
      >
        <animate
          attributeName="opacity"
          begin="-1.461038961038961s"
          dur="2.272727272727273s"
          keyTimes="0;1"
          repeatCount="indefinite"
          values="1;0"
        />
      </rect>
    </g>
    <g
      transform="rotate(128.57142857142858 50 50)"
    >
      <rect
        fill="#f5deb3"
        height="3"
        rx="1.5"
        ry="1.5"
        width="3"
        x="48.5"
        y="39.5"
      >
        <animate
          attributeName="opacity"
          begin="-1.2987012987012987s"
          dur="2.272727272727273s"
          keyTimes="0;1"
          repeatCount="indefinite"
          values="1;0"
        />
      </rect>
    </g>
    <g
      transform="rotate(154.28571428571428 50 50)"
    >
      <rect
        fill="#f5deb3"
        height="3"
        rx="1.5"
        ry="1.5"
        width="3"
        x="48.5"
        y="39.5"
      >
        <animate
          attributeName="opacity"
          begin="-1.1363636363636362s"
          dur="2.272727272727273s"
          keyTimes="0;1"
          repeatCount="indefinite"
          values="1;0"
        />
      </rect>
    </g>
    <g
      transform="rotate(180 50 50)"
    >
      <rect
        fill="#f5deb3"
        height="3"
        rx="1.5"
        ry="1.5"
        width="3"
        x="48.5"
        y="39.5"
      >
        <animate
          attributeName="opacity"
          begin="-0.974025974025974s"
          dur="2.272727272727273s"
          keyTimes="0;1"
          repeatCount="indefinite"
          values="1;0"
        />
      </rect>
    </g>
    <g
      transform="rotate(205.71428571428572 50 50)"
    >
      <rect
        fill="#f5deb3"
        height="3"
        rx="1.5"
        ry="1.5"
        width="3"
        x="48.5"
        y="39.5"
      >
        <animate
          attributeName="opacity"
          begin="-0.8116883116883117s"
          dur="2.272727272727273s"
          keyTimes="0;1"
          repeatCount="indefinite"
          values="1;0"
        />
      </rect>
    </g>
    <g
      transform="rotate(231.42857142857142 50 50)"
    >
      <rect
        fill="#f5deb3"
        height="3"
        rx="1.5"
        ry="1.5"
        width="3"
        x="48.5"
        y="39.5"
      >
        <animate
          attributeName="opacity"
          begin="-0.6493506493506493s"
          dur="2.272727272727273s"
          keyTimes="0;1"
          repeatCount="indefinite"
          values="1;0"
        />
      </rect>
    </g>
    <g
      transform="rotate(257.14285714285717 50 50)"
    >
      <rect
        fill="#f5deb3"
        height="3"
        rx="1.5"
        ry="1.5"
        width="3"
        x="48.5"
        y="39.5"
      >
        <animate
          attributeName="opacity"
          begin="-0.487012987012987s"
          dur="2.272727272727273s"
          keyTimes="0;1"
          repeatCount="indefinite"
          values="1;0"
        />
      </rect>
    </g>
    <g
      transform="rotate(282.85714285714283 50 50)"
    >
      <rect
        fill="#f5deb3"
        height="3"
        rx="1.5"
        ry="1.5"
        width="3"
        x="48.5"
        y="39.5"
      >
        <animate
          attributeName="opacity"
          begin="-0.3246753246753247s"
          dur="2.272727272727273s"
          keyTimes="0;1"
          repeatCount="indefinite"
          values="1;0"
        />
      </rect>
    </g>
    <g
      transform="rotate(308.57142857142856 50 50)"
    >
      <rect
        fill="#f5deb3"
        height="3"
        rx="1.5"
        ry="1.5"
        width="3"
        x="48.5"
        y="39.5"
      >
        <animate
          attributeName="opacity"
          begin="-0.16233766233766234s"
          dur="2.272727272727273s"
          keyTimes="0;1"
          repeatCount="indefinite"
          values="1;0"
        />
      </rect>
    </g>
    <g
      transform="rotate(334.2857142857143 50 50)"
    >
      <rect
        fill="#f5deb3"
        height="3"
        rx="1.5"
        ry="1.5"
        width="3"
        x="48.5"
        y="39.5"
      >
        <animate
          attributeName="opacity"
          begin="0s"
          dur="2.272727272727273s"
          keyTimes="0;1"
          repeatCount="indefinite"
          values="1;0"
        />
      </rect>
    </g>
  </svg>
</section>
`;
